src/descriptions.js

```diff
@@ -81,7 +81,9 @@ module.exports = {
   "ghost lady": "At night you can visit another player. If that player gets attacked, they will be protected and your role will be revealed to them. You cannot visit after protecting once. Your aura is **Good** and you belong to the **Village** team.",
   baker: "Each night you can select a player to give them bread, giving them an extra vote for the next day. Your aura is **Good** and you belong to the **Village** team.",
   "split wolf": "During the first three nights, you can choose to bind yourself to another player. If you do this, you will count double to the werewolf win condition. However, if one of you is killed, the other will also die. Your aura is **Evil** and you belong to the **Werewolves** team.",
-   vigilante: "During the day you can either shoot gunner  or reveal loudmouth  another player. Both actions can only be done once, but not during the same day. Your aura is **Unknown** and you belong to the **Village** team.",
+   vigilante: "During the day you can either shoot like gunner  or reveal like loudmouth  another player. Both actions can only be done once, but not during the same day. Your aura is **Unknown** and you belong to the **Village** team.",
```

Comment:
There are some double spaces in between some of these words, instead of just one space